deployment/sriov-network-operator/values.yaml

```diff
@@ -0,0 +1,23 @@
+operator:
+  tolerations:
+    - key: "node-role.kubernetes.io/master"
+      operator: "Exists"
+      effect: "NoSchedule"
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  nameOverride: ""
+  fullnameOverride: ""
+  resourcePrefix: "openshift.io"
+  enableAdmissionController: false
+  cniBinPath: "/opt/cni/bin"
+  clusterType: "kubernetes"
+
+# Image URIs for sriov-network-operator components
+images:
+  operator: quay.io/openshift/origin-sriov-network-operator
```

Comment:
It looks good to me. But we shall use the images build for vanilla k8s here. So I suggest we wait until #83 merged.